### ISSUE TITLE
Categorizer is not working reliable

### ISSUE THREAD

**Issue by reloxx13** [Tensai75/nzb-monkey-go#18]
The categorizer is not working reliable, it looks like a race-condition and sometimes movie regex wins before the series regex.    

example:       
`XYZ.S17E26.Ger.Eng.AAC.720p.WebDL.x264-XYZ-UVW`       
    
First time it was matched as movie, then I add it again, and now it's correctly matched as series. Mostly happens if I click multiple lnk links quickly one after the other and multiple nmg´s console windows open.     

Regexes (Worked fine like forever in nzbdonkey):    
```
[CATEGORIZER]
# Place your category and you regex here
# Please uncomment the following lines
series = "(s\d+e\d+|s\d+ complete)"
4k = "(2160p)"
movies = "(x264|xvid|bluray|720p|1080p|untouched)"
games = "(Z|Y|X)"
```

**Comment by reloxx13** [comment]
![grafik](https://github.com/Tensai75/nzb-monkey-go/assets/14855001/980c814f-8f70-4638-99f3-b2e6a7338a02)

**Comment by Tensai75** [comment]
Fixed with [v0.1.8](https://github.com/Tensai75/nzb-monkey-go/releases/tag/v0.1.8) (https://github.com/Tensai75/nzb-monkey-go/commit/51f2dfe3bc8abe6afcb1da5e9923153bfc6ef4e5)
